py/occam_base/model.py

```diff
@@ -15,24 +21,30 @@ def __init__(self, ref=None):
         self._id = 0
 
     @property
-    def id_(self):
+    def id_(self) -> int:
         return self._id
 
     @id_.setter
-    def id_(self, id_):
+    def id_(self, id_: int) -> None:
         self._id = id_
 
     @property
-    def fit_table(self):
+    def fit_table(self) -> None:
         pass
 
     @fit_table.setter
-    def fit_table(self, model):
-        self.ref.makeFitTable(model)
+    def fit_table(self, model) -> None:
+        self._ref.makeFitTable(model)
 
     @fit_table.deleter
     def fit_table(self):
-        self.ref.deleteFitTable()
+        self._ref.deleteFitTable()
 
     def set(self, **kwargs):
         pass
+
+    def get_print_name(self) -> str:
```

Comment:
Could we use @properties here as well?

Reply:
I left it out without property for the reasons we discussed before, regarding it crashing when the object is null and no checks are performed on c++ side.

Reply:
I'll modify the pyocam.cpp to fix that and re commit